src/com/lftechnology/java/training/alina/collections/generics/wildcards/WildcardDemo.java

```diff
@@ -0,0 +1,94 @@
+
+package com.lftechnology.java.training.alina.collections.generics.wildcards;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Write a method which takes List of any subclass of object Customer. (use
+ * wildcard)
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class WildcardDemo {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)